pages/index.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Introduction
+---
+
+# Introduction
+
+Welcome to the Liberty Protocol specification!
+Here you can find a detailed documentation regarding the current state of the protocol, previous iterations and proposals for future extensions.
+**Please note, the protocol specification is still very much a work in progress at this time, and the information within is subject to frequent changes.**
+
+# Goals & Purpose
+
+The free communication of users on the internet faces a variety of problems in the modern day.
+These challenges include censorship from state and corporate actors, the amplification of misinformation through viral content, and an ever-shrinking collection of near monopolies with absolute power over social interaction in the 21st century.
+Through the Liberty Protocol, we hope mitigate and ideally solve these challenges in the way social interaction operates online.
+
+## Censorship
+
+TBD - Quick summary on the threat of censorship online; maybe Harry has some suggestions
+
+## Misinformation
+
+TBD - Quick summary on the threat of misinformation; ask Shannon for input
```

Comment:
I really think these items don't belong in a spec, which is aimed at helping developers use the platform.  At most they should link to outside articles, such as the blog site.

Reply:
I'm just going on the feedback from the OTS meeting. I think the general consensus was that we wanted to briefly mention this stuff here to drive home the point, but the bulk of this writing will be on the developer portal.

Reply:
Ah! Ok. I either missed that entirely or just forgot about it.